instrumentation/khttp-0.1/src/main/java/io/opentelemetry/auto/instrumentation/khttp/KHttpAdvice.java

```diff
@@ -32,14 +32,16 @@
 import khttp.KHttp;
 import khttp.responses.Response;
 import net.bytebuddy.asm.Advice;
+import net.bytebuddy.implementation.bytecode.assign.Assigner.Typing;
 
 public class KHttpAdvice {
 
   @Advice.OnMethodEnter(suppress = Throwable.class)
   public static SpanWithScope methodEnter(
       @Advice.Argument(value = 0) String method,
       @Advice.Argument(value = 1) String uri,
-      @Advice.Argument(value = 2) Map<String, String> headers) {
+      @Advice.Argument(value = 2, typing = Typing.DYNAMIC, readOnly = false)
+          Map<String, String> headers) {
```

Comment:
I don't think `typing = Typing.DYNAMIC` is needed since the parameter type `Map` is the same as the type in the matcher `takesArgument(2, named("java.util.Map"))`